elements/pf-date-picker/demo/pf-date-picker.html

```diff
@@ -0,0 +1,75 @@
+  </section>
+
+  <section>
+    <h2>Localization</h2>
+    <pre><code>&lt;pf-date-picker localizationLanguageCode="fi"&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <pf-date-picker localizationLanguageCode="fi"></pf-date-picker>
+    <p>The locale string can be passed to the date picker to set the date format.</p>
+    <p>In the above given example, the locale is set to <span>Finnish: fi</span></p>
+    <p>The date format is set globally, independent of the user's locale. 
+      This means that all users will see the same date format, regardless of their language or region.</p>
+  </section>
+
+  <section>
+    <h2>Translation</h2>
+    <pre><code>&lt;pf-date-picker translationLanguageCode="fr"&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <pf-date-picker translationLanguageCode="fr"></pf-date-picker>
+    <p>The translation language string can be passed to the date picker to localize the month names.</p>
+    <p>In the above given example, the language is set to <span>French: fr</span></p>
+  </section>
+
+  <section>
+    <h2>Set Date</h2>
+    <pre><code>&lt;pf-date-picker inputDate=${new Date(2023, 0, 1)}&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <!-- ***** The value for inputDate must be passed as **DATE TYPE** from parent component (Here the value is passed as string type 
+      for demo purpose, as there are some restrictions for adding Date type value in .html file). -->
+    <pf-date-picker inputDate="Sun Jan 01 2023 05:30:00 GMT+0530 (India Standard Time)"></pf-date-picker>
+    <p>The default date value can be passed to the date picker to set the initial date that is displayed.</p>
+    <p>In the above given example, the date set to be displayed initially is <span>January 1, 2023</span>.</p>
+  </section>
+
+  <section>
+    <h2>Set Min and Max Date</h2>
+    <pre><code>&lt;pf-date-picker minDate=${new Date(2023, 0, 2)} minDate=${new Date(2023, 0, 20)} 
+      inputDate=${new Date(2023, 0, 3)}&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <!-- ***** The values for minDate and maxDate must be passed as **DATE TYPE** from parent component (Here the values are passed as string type 
+      for demo purpose, as there are some restrictions for adding Date type value in .html file). -->
+    <pf-date-picker 
+      inputDate="Tue Jan 03 2023 05:30:00 GMT+0530 (India Standard Time)"
+      minDate="Mon Jan 02 2023 00:00:00 GMT+0530 (India Standard Time)"
+      maxDate="Fri Jan 20 2023 00:00:00 GMT+0530 (India Standard Time)">
+    </pf-date-picker>
+    <p>The minimum and maximum valid dates can be passed to the date picker to restrict the range of dates that can be selected</p>
+    <p>In the above given example, the minimum valid date 
+      is <span>January 2, 2023</span>, and the maximum valid date is <span>January 20, 2023</span></p>
+  </section>
+
+  <section>
+    <h2>Disabled</h2>
+    <pre><code>&lt;pf-date-picker isDisabled="true"&gt;&lt;/pf-date-picker&gt;</code></pre>
+    <pf-date-picker isDisabled="true"></pf-date-picker>
```

Comment:
We typically remove the `is` prefix for boolean attributes, see `<pf-button disabled>` for example

Reply:
Removed the `is` prefix form the disabled property.